packages/browser/src/Widget/Server/SPARQL/View/SparqlQueryWindow.js

```diff
@@ -31,12 +27,17 @@ export class SparqlQueryWindow extends EventSender {
    *                                              pairs. The keys of these pairs should correspond
    *                                              with the cases in the updateDataView() function.
    */
-  constructor(sparqlProvider, itownsView, configSparqlWidget) {
+  constructor(
+    sparqlProvider,
+    itownsView,
+    configSparqlWidget,
+    widgetId = '_window_sparqlQueryWindow'
```

Comment:
id should not used [see](https://github.com/VCityTeam/UD-Viz/issues/567) let it this way I'll fix it later, just for your information